plugin-testweaver.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "testweaver"
+github: "jenkinsci/testweaver-plugin"
+paths:
+- "org/jenkins-ci/plugins/testweaver"
```

Comment:
Mismatch with https://github.com/jenkinsci/testweaver-plugin/blob/master/pom.xml#L10

Reply:
Done! I have modified pom.xml file.

Reply:
The file plugin-testweaver.yml should be in repository-permissions-updater? Why is not in repository-permissions-updater/permissions?

Reply:
@ciprianmorosanu yes, you want to file a followup PR to fix this. I do not think it will be read in any way, being in the wrong location.

Reply:
Missed that 😦  It's in the checklist for a reason…

Reply:
What can I do?

Reply:
Create a followup PR moving the file.